packages/expo/src/launch/registerRootComponent.tsx

```diff
@@ -1,7 +1,17 @@
 import 'expo/build/Expo.fx';
+import { activateKeepAwake } from 'expo-keep-awake';
 import * as React from 'react';
 import { AppRegistry } from 'react-native';
 
-export default function registerRootComponent<P>(component: React.ComponentType<P>): void {
-  AppRegistry.registerComponent('main', () => component);
+import withExpoRoot from './withExpoRoot';
+import { InitialProps } from './withExpoRoot.types';
+
+if (__DEV__) {
+  activateKeepAwake();
+}
```

Comment:
This is a side effect -- we should change this file to .fx.tsx or move it into registerRootComponent.

Reply:
Added a comment to do this in another PR.